test/extensions/transport_sockets/tls/test_data/i1_cert.pem

```diff
@@ -0,0 +1,24 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Do you happen to know why this file (and a few others that follow) was empty before?

Reply:
```
generate_cert_chain() {
    local certname
    local ca_name="${1}"
    rm test_long_cert_chain.pem
    touch test_long_cert_chain.pem
    for x in {1..4}; do
        certname="i$x"
        if [[ $x -gt 1 ]]
        then
            ca_name="i$((x - 1))"
        fi
        echo "$x: $certname $ca_name"
        generate_ca $certname $ca_name
    done
    for x in {1..3}; do
        cat "i${x}_cert.pem" >> test_long_cert_chain.pem
    done
    mv i4_cert.pem test_random_cert.pem
}
```
They seems like the intermediate files that used to generate  `test_long_cert_chain.pem` and `test_random_cert.pem`.

Seems that the `certs.sh` didn't clean these files.

Reply:
Should I clean the files directly or clean these files and update `certs.sh` in new separated PR?

Reply:
All these files are deleted and I will update the `certs.sh` with new PR.